transports/http_admin.go

```diff
@@ -316,3 +317,48 @@ func (h *TransportHTTP) AdminGetSharedConfig(ctx context.Context) (*models.Share
 
 	return model, nil
 }
+
+func (h *TransportHTTP) AdminGetContacts(ctx context.Context, conditions map[string]interface{}, metadata *models.Metadata, queryParams *QueryParams) ([]*models.Contact, ResponseError) {
+	jsonStr, err := json.Marshal(map[string]interface{}{
+		FieldConditions:  conditions,
+		FieldMetadata:    processMetadata(metadata),
+		FieldQueryParams: queryParams,
+	})
+	if err != nil {
+		return nil, WrapError(err)
+	}
+
+	var contacts []*models.Contact
+	err = h.doHTTPRequest(ctx, http.MethodPost, "/admin/contact/search", jsonStr, h.adminXPriv, true, &contacts)
+	return contacts, WrapError(err)
+}
+
+func (h *TransportHTTP) AdminUpdateContact(ctx context.Context, id, fullName string, metadata *models.Metadata) (*models.Contact, ResponseError) {
+	jsonStr, err := json.Marshal(map[string]interface{}{
+		"fullName":    fullName,
+		FieldMetadata: processMetadata(metadata),
+	})
+	if err != nil {
+		return nil, WrapError(err)
+	}
+	var contact *models.Contact
+	err = h.doHTTPRequest(ctx, http.MethodPatch, fmt.Sprintf("/admin/contact/%s", id), jsonStr, h.adminXPriv, true, &contact)
```

Comment:
Maybe the `&` is not necessary in `&contact`, because the `contact` variable is already a pointer 🤔.

Please, check all occurrences of this.

Reply:
No, this part `&contact` is needed try to removing it and ran my test u will see empty object gets returned, i can remove pointer here `*models.Contact` if u think this is going to look cleaner ?

Reply:
If it works, don't change it 🙈